test/utilities.ts

```diff
@@ -82,3 +85,30 @@ export const applyBalances = async function (
     await epochTokenLocker.withdraw(userAddress, tokenAddress);
   }
 };
+
+/**
+ * Makes deposit transactions from a list of Deposit Objects
+ * @param {number} numTokens - number of tokens to be registered on this exchange.
+ * @param {number} maxTokens - Maximum number of tokens (a contract contructor parameter)
+ * @returns {}
```

Comment:
I think the return type here is a `Promise<BatchExchangeInstance>` unless I misread something.